pkgs/development/libraries/mtxclient/default.nix

```diff
@@ -18,8 +30,18 @@ stdenv.mkDerivation rec {
     "-Dnlohmann_json_DIR=${nlohmann_json}/lib/cmake/nlohmann_json"
   ];
 
-  nativeBuildInputs = [ cmake pkgconfig ];
-  buildInputs = [ boost openssl zlib libsodium olm ];
+  nativeBuildInputs = [
+    cmake
+    pkgconfig
+  ];
+  buildInputs = [
+    spdlog
```

Comment:
Commit message says "mtxclient: format inputs and arguments", but you add `spdlog`.